src/runtime_src/core/common/query_requests.h

```diff
@@ -352,6 +353,23 @@ struct pcie_device : request
   }
 };
 
+struct pcie_revision : request
```

Comment:
There are a few places where XRT depends on `pcie_device` as being the only identifier for the system. By adding in an additional value we throw a lot of that logic away. We need a better way to get the device ID that includes the revision number...

Instead of creating a new query for the revision, perhaps we can create a new query that creates a pcie identifier? Whatever change we make in XRT would need to update all usages of `pcie_device` so this is not a little update.

Reply:
@maxzhen  I just did that PCIe identifier key: "pci device id and revision id combined" in our driver. 
I added the pci_revision here in XRT module to let shim_tests run without any issues.

As per Daniel comments looks like if we want to include revision id as the PCIE identifier it impacts multiple places in XRT module. So can this be taken care of by the XRT team?
I don't have much exposure to this XRT module and don't wanna cause lot of regressions.